src/transactions.cpp

```diff
@@ -0,0 +1,22 @@
+#pragma once
+#include "C:\Users\Hamza Rashid\Desktop\C++\Trading Engine\include\TradingEngine\transactions.h"
+
+#include <string>
+using std::string;
+
+#include <vector>
+using std::vector;
+
+/* Size of total modified order's */
+size_t Transactions::getLength() const {
+	return raw_orders.size();
+}
+
+/* Size of executed order's */
+size_t Transactions::getExecutedOrders() const {
```

Comment:
The name is not reflecting what you are returning. Rename to `getNumberOfExecutedOrders`

Reply:
done.